server/bin/dev.js

```diff
@@ -3,7 +3,7 @@ const routes = require('../routes');
 const pages = require('../routes/pages');
 const tests = require('../../test/frontend/routes');
 const express = require('express');
-const expressWs = require('express-ws');
+const expressWs = require('@dannycoates/express-ws');
```

Comment:
Are there changes in your copy of express-ws that are needed? Or was this just for debugging?

Reply:
I updated the `ws` dependency to the latest major version. We've been using a fork of express-ws since the beginning, I just finally pushed it to npm